src/test/java/dk/cngroup/lentils/service/TeamServiceIntegrationTest.java

```diff
@@ -30,13 +30,21 @@
     @Autowired
     private UserRepository userRepository;
 
+    private int numOfUsersBeforeTest;
+    private int numOfTeamsBeforeTest;
+
+    @Before
+    public void setup() {
+        numOfUsersBeforeTest = userRepository.findAll().size();
+        numOfTeamsBeforeTest = teamRepository.findAll().size();
+    }
+
     @Test
     public void addingOneTeamCorrectlyCreatesUser() {
         createAndSaveTeam("pump", 5, "8888");
-        assertEquals(1, teamService.getAll().size());
-        assertEquals(3, userRepository.findAll().size());
-        assertNotEquals(userRepository.findByUsername("pump").get().getUserId(),
-                teamRepository.findByName("pump").getTeamId());
```

Comment:
Mám pocit, že byl problém s tím, že user a team se vytvářeli se stejným ID, proto jsem tam dával tyto assertNotEquals :)

Reply:
OK, ale porad nevim, cemu to vadi. Spravne by nemelo.

Reply:
Nebo nevim, zatim nechavam PR otevreny komentum.
 Jeste se zeptam, co kdyz ma jakykoliv uzivatel stejne ID jako nejaky i jiny team? Vadi to necemu?

Reply:
Anebo to souvisi s tim, jak Hibernate pouziva jednu sekvenci pro vsechny tabulky v DB?

Reply:
Tak tohle bude spíš otázka na Martina :D mám totiž pocit, že mi to říkal on @mjanousek

Reply:
Problem byl jiny. Stavalo se to, ze:

1. chteli jste vytvorit team
2. vytvoril se team napr. s id 1
3. zaroven s tim se vygeneroval a ulozit user s id napr 3
4. ale pak se nenastavila vazba s userem s id 3 ale s userem s id 1 (Tj. vzdy se stejnym id, ktere mel team)

Reply:
Aha, tak to jsem si to špatně pamatoval, moje chyba.

Reply:
Aha, chápu. I tak ale ten assert nebyl správně podle mě si myslím.
Takže, chcete ho tam nechat nebo nějak opravit?

Reply:
Vrátil jsem zpět, jen upravil logiku.
Teď by test měl kontrolovat, jestli má po uložení team přirazeného správného uživatele a naopak.